src/process.cpp

```diff
@@ -424,6 +428,15 @@ std::optional<proc::proc_t> parse(const std::string &file_name) {
         ctx.image_path = parse_env_val(this_env, *image_path);
       }
 
+      if(id) {
+        ctx.id = parse_env_val(this_env, *id);
+      }
+      else {
+        ctx.id = std::to_string(app_index);
+      }
+      // Always increment index to avoid order shuffling in moonlight
```

Comment:
@cgutman For the two clients I tested (iOS/QT), it looks like app ordering is actually alphabetical, not determined by ID. Does that sound correct?

Reply:
Yeah, the apps are displayed in alphabetical order, but the the cached box art on disk is stored by ID.

If you go into the cache folder, you'll see the box art there named `<APP ID>.png`:
![image](https://user-images.githubusercontent.com/2695644/210277351-d3023ee3-9f3a-4edf-b7f1-e53fd88487f2.png)